File: Python-Fundamentals/Labs-And-Homeworks/Functions/perfect_number.py
def is_perfect(some_number: int) -> str:
    divisors_sum = 0
    for divisor in range(1, some_number):
        if some_number % divisor == 0:
            divisors_sum += divisor
    if some_number == divisors_sum:
        return "We have a perfect number!"
    return "It's not so perfect."


number = int(input())
print(is_perfect(number))

File: JS-Front-End/JS-Front-End/Labs-And-Homeworks/Functions-and-Statements/Exercise/perfect-number.js
function isPerfectNumber(num) {

    let sum = 1;

    for ( let i = 2; i < num; i++) {
        if (num % i === 0) sum += i;
    }

    if (num === sum) {
        console.log('We have a perfect number!');
    } else {
        console.log("It's not so perfect.");
    }
}

// Test Case

isPerfectNumber(6);
isPerfectNumber(28);
isPerfectNumber(1236498);